data/core/commands/doc.lua

```diff
@@ -113,10 +125,17 @@ local commands = {
     -- If the clipboard has changed since our last look, use that instead
     if doc().cursor_clipboard["full"] ~= clipboard then
       doc().cursor_clipboard = {}
+      doc().cursor_clipboard_whole_line = {}
     end
     for idx, line1, col1, line2, col2 in doc():get_selections() do
```

Comment:
What if we paste the whole clipboard on each cursor if their number is different than when we used copy?
For example: we used 3 cursors to copy lines; if on paste we have 3 cursors, we paste each line on its corresponding cursor; if on paste we have 2 cursors, we paste *all* 3 lines on each cursor.